source/_docs/z-wave/controllers.markdown

```diff
@@ -73,32 +73,32 @@ Firmware can be upgraded using the below directions:
   - Z-Wave.Me Razberry 2 (500 series)
   - Z-Wave.Me Razberry 1 (300 series)
 
-If you are just starting out, we recommend that you purchase a 700 series controller or a Raspberry Pi module. The 700 series controllers are the more recent version (when compared to the 500 series). The 700 series controllers support SmartStart, which allows you to add a device by scanning a QR code.
+If you are just starting out, we recommend that you purchase a 700 series adapter or a Raspberry Pi module. The 700 series adapters are the more recent version (when compared to the 500 series). The 700 series adapters support SmartStart, which allows you to add a device by scanning a QR code.
```

Comment:
Actually, I think this entire sentence needs to be reworked:

- We recommend starting with 800 series now, especially in EU because of the upcoming Long Range support
- 700 series only supports Long Range in US (with one exception)
- 700 and 800 series (and some recent 500 series firmwares) support SmartStart

But this is too complex to explain here, so we should just say that 800 series is the most future proof and has the best RF performance.